sdk/core/core/inc/az_http_response.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef AZ_HTTP_RESPONSE_H
+#define AZ_HTTP_RESPONSE_H
+
+#include <az_mut_span.h>
+
+#include <_az_cfg_prefix.h>
+
+typedef struct {
+  az_mut_span value;
+} az_http_response;
+
+AZ_NODISCARD AZ_INLINE az_result
+az_http_response_init(az_http_response * const self, az_mut_span const * const value) {
```

Comment:
`az_mut_span` should passed by value in this case.
```suggestion
az_http_response_init(az_http_response * const self, az_mut_span const value) {
```